turn/__init__.py

```diff
@@ -208,15 +208,15 @@ def do_request(self, data=None):
             # Only send data if passed in, but if it's a query that
             # needs the data field, then auth has to be in here and not params
             data=data if data is not None else None,
-            params=self.params() if data is None else None
+            params=self.params() if data is None else None,
         )
 
 
 class TurnMessageTemplates(TurnBusinessManagementRequest):
     endpoint_name = "message_templates"
 
     def get_message_templates(self):
-        self.method = 'GET'
+        self.method = "GET"
```

Comment:
Does your IDE do this or do you just really dislike single quotes?

Reply:
I suppose both could be true.

Reply:
I think I did this mostly by accident. I've started running `black` on new projects to maintain some consistency, and didn't notice that this was collateral damage in this commit.